code/game/objects/items/tools/powertool.dm

```diff
@@ -0,0 +1,86 @@
+//Power Tools
+/obj/item/powertool
+	var/stateswap = 0 //what state is it in
+	var/icon_state_swap = 0 //which icon should it be
+	var/tool_behaviour_swap = 0 //what tool behaviour(s) should be here
+	var/swapsound = 0 //the sound that plays for swapping
+
+/obj/item/powertool/attack_self(mob/living/carbon/user) //activating the item causes it to swap icons and behaviours
+	stateswap = !stateswap
+	if(stateswap)
+		icon_state = icon_state_swap
+		tool_behaviour = tool_behaviour_swap
+	else
+		icon_state = initial(icon_state)
+		tool_behaviour = initial(tool_behaviour)
+	playsound(get_turf(user), swapsound, 50, 1)
+	to_chat(user, "<span class='notice'>You attach a different bit to [src].</span>")
+	return
+		
+/obj/item/powertool/drill
+	name = "Power Drill"
+	desc = "A simple hand drill with bolt and screw bits."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "drill_screw"
+	item_state = "drill"
+	icon_state_swap = "drill_bolt"
+	hitsound = 'sound/items/drill_hit.ogg'
+	usesound = 'sound/items/drill_use.ogg'
+	swapsound = 'sound/items/change_drill.ogg'
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	force = 5
+	throwforce = 7
+	w_class = WEIGHT_CLASS_SMALL
+	materials = list(MAT_METAL=150,MAT_SILVER=50,MAT_TITANIUM=25)
+	origin_tech = "engineering=1;combat=1"
+	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked")
+	toolspeed = 0.25
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
+	tool_behaviour = TOOL_SCREWDRIVER
+	tool_behaviour_swap = TOOL_WRENCH
+
+/obj/item/powertool/drill/suicide_act(mob/user)
+	if(tool_behaviour == TOOL_WRENCH)
+		user.visible_message("<span class='suicide'>[user] is pressing [src] against [user.p_their()] head! It looks like [user.p_theyre()] trying to commit suicide!")
+	else
+		user.visible_message("<span class='suicide'>[user] is putting [src] to [user.p_their()] temple. It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	return BRUTELOSS
+
+/obj/item/powertool/powerjaws
+	name = "jaws of life"
+	desc = "A set of jaws of life, the magic of science has managed to fit it down into a device small enough to fit in a tool belt. has prying and cutting attachments."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "jaws_pry"
+	item_state = "jawsoflife"
+	icon_state_swap = "jaws_cutter"
+	usesound = 'sound/items/jaws_pry.ogg'
+	swapsound = 'sound/items/change_jaws.ogg'
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
```

Comment:
I think it might be a good idea to move `SLOT_BELT` and `WEIGHT_CLASS_SMALL` to the base `/obj/item/powertool` class (but I don't feel too strong about this)